src/main/java/bio/terra/pdao/bigquery/BigQueryPdao.java

```diff
@@ -489,15 +565,16 @@ private boolean existenceCheck(String name) {
         }
     }
 
-    private String prefixName(String name) {
+    public String prefixName(String name) {
```

Comment:
I think this is public due to the code in the step. I'd rather that code be in the pdao so this doesn't have to get exposed.